index.html

```diff
@@ -93,6 +124,30 @@ <h1 class="projects">
                      </div>
                     </div>
             </div>
+            <div class="image-dots">
+                <img src="./images/image_dots.png
+                " alt="image-dots">
+            </div>
+            <div class="geo">
+                <img src="images/geometry_2.png" alt="geometry">
+            </div>
+            <div class="geo2">
+                <img src="./images/geo.png" alt="Geometry">
+            </div>
+             <div class="img-shape">
+                <img src="./images/Geo2.png" alt="square">
+            </div>
+            <div class="img-geo3">
+                <img src="./images/Geo3.png" alt="Shape">
+            </div>
+            <div class="img-dots">
+                <img src="./images/img-dots.png" alt="image">
+            </div><div class="img-geo4">
+                <img src="./images/Geo4.png" alt="geometry">
+            </div>
+            <div class="img-geo5">
+                <img src="./images/Geo5.png" alt="  geo ">
```

Comment:
I noticed your web application has no about section it would be nice if you added one. Again pay close attention to the figma design.

Your design: 

![Screenshot from 2022-07-15 22-35-22](https://user-images.githubusercontent.com/79506105/179289393-a78293c2-48b3-4ea9-9040-1e5cdb47c475.png)

Figma design:


![Screenshot from 2022-07-15 22-21-16](https://user-images.githubusercontent.com/79506105/179289468-c0621635-76c2-4179-b92e-2f55935727d8.png)
Please align your content in the proper position.